reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -8,6 +8,11 @@
 
 <header id="liveupdate-header">
   <h1 id="liveupdate-title">${thing.event.title}</h1>
+  % if thing.event.description:
+  <div id="liveupdate-description">
```

Comment:
Bold, italics, etc. are quite useful. Either way, the markdown styles (for lack of a better name) make text way more readable than the ugly default reddit comes with.